__tests__/components/Badge/Badge.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import { Badge, Button } from '../../../src';
+
+import '@testing-library/jest-dom';
+
+let container: HTMLDivElement;
+
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  document.body.removeChild(container);
+  container = null;
+});
+
+describe('Badge', () => {
+  it('should render a deafult badge', () => {
```

Comment:
'default'

Reply:
Resolved in b4839d276f108d54554be70a818e7d25911e1a68